packages-next/fields/src/types/file/views/Field.tsx

```diff
@@ -22,44 +22,53 @@ export function validateRef({ ref }: { ref: string }) {
 }
 
 const RefView = ({
+  field,
   onChange,
   onCancel,
   error,
 }: {
+  field: any;
   onChange: (value: string) => void;
   onCancel: () => void;
   error?: string;
 }) => {
   return (
-    <Stack
-      gap="small"
-      across
-      css={{
-        width: '100%',
-        justifyContent: 'space-between',
-        'div:first-of-type': {
-          flex: '2',
-        },
-      }}
-    >
-      <TextInput
-        placeholder="Paste the file ref here"
-        onChange={event => {
-          onChange(event.target.value);
-        }}
+    <Fragment>
+      <VisuallyHidden as="label" htmlFor={`${field.path}--ref-input`}>
+        Paste the file ref here
+      </VisuallyHidden>
+      <Stack
+        gap="small"
+        across
         css={{
           width: '100%',
+          justifyContent: 'space-between',
+          'div:first-of-type': {
```

Comment:
which element is this targeting(and the one in the image field) and can you have the style the closer to where it is?

Reply:
@mitchellhamilton not a code addition from this PR, (highlighted because we moved some code around above), however I'd hazard a guess that this is here because our stack component wraps all children in a partitioning div (since we couldn't rely on `gap` for all browsers at the time of creating that component), *looks* like its targeting the textinput

Reply:
oh ok, nvm then.